### PR TITLE
Make Accordion Usable for External Projects

### PR DESCRIPTION
**Pull request by synbot**
### Acceptance Criteria
1. index.js is updated to export the component from the /dist folder
2. Readme documentation is updated
3. Component can be built and tested

### CODE REVIEW

**Comment by kylecesmat** [comment]
:m: :star:

File: src/accordion.jsx [hunk 1]
```diff
@@ -12,24 +12,25 @@ module.exports = React.createClass({
...
-            message : React.PropTypes.string
+            title          : React.PropTypes.string,
+            message        : React.PropTypes.string,
+            panelClassName : React.PropTypes.string
         })),
+        toggleSpeed  : React.PropTypes.number,
         toggleHeight : React.PropTypes.number,
-        className    : React.PropTypes.string
+        customClass  : React.PropTypes.string
     },
```
Comment (natestormer) [inline src/accordion.jsx:22]: probably should change this to className

File: src/accordion.jsx [hunk 2]
```diff
@@ -138,7 +134,7 @@ module.exports = React.createClass({
     {
         var containerStyle = [
             'accordion__wrapper',
-            this.props.className
+            this.props.customClass
```
Comment (natestormer) [inline src/accordion.jsx]: this.props.className

**Comment by natestormer** [comment]
:interrobang: just those 2 small things

**Comment by bobeagan** [comment]
:+1:

**Pull request merged**